File: auth-server/src/main/java/example/demo/AuthServer.java
package example.demo;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.cloud.client.discovery.EnableDiscoveryClient;
import org.springframework.web.reactive.config.EnableWebFlux;


@EnableWebFlux
@EnableDiscoveryClient
@SpringBootApplication
public class AuthServer {

    public static void main(String[] args) {
        SpringApplication.run(AuthServer.class, args);
    }
}

File: auth-server/src/main/kotlin/example/demo/AuthServer.kt
package example.demo;

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.autoconfigure.data.mongo.MongoReactiveDataAutoConfiguration
import org.springframework.boot.runApplication
import org.springframework.cloud.client.discovery.EnableDiscoveryClient


@EnableDiscoveryClient
@SpringBootApplication(exclude = [MongoReactiveDataAutoConfiguration::class])
class AuthServer

fun main(args: Array<String>) {
    runApplication<AuthServer>(*args)
}